src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/requestaccess/CreateDataAccessSubmissionStep2.java

```diff
@@ -112,18 +112,12 @@ public void invoke(FileUpload fileUpload) {
 		peopleSuggestWidget.addItemSelectedHandler(new CallbackP<SynapseSuggestion>() {
 			public void invoke(SynapseSuggestion suggestion) {
 				peopleSuggestWidget.clear();
-				CreateDataAccessSubmissionStep2.this.accessorsList.addUserBadge(suggestion.getId());
+				AccessorChange change = new AccessorChange();
+				change.setUserId(suggestion.getId());
+				change.setType(AccessType.GAIN_ACCESS);
+				CreateDataAccessSubmissionStep2.this.accessorsList.addUserBadge(change);
```

Comment:
please consider renaming:
accessorsList to accessorsChangeList
addUserBadge to addAccessorChange